aces/Loris
# [user_accounts] User with only superuser permission selected can edit other user's permissions

## Brief summary of changes
If a user has superuser permission but no other permissions individually selected, they were not able to edit the permissions of another user. This is fixed by adding an override in the `_process` function of the edit_user class which checks if a user is a superuser before removing any permissions that the superuser "does not have" from the list of edited permissions.

#### Testing instructions (if applicable)

1. Create a new user and select the superuser permission but not any other permissions specifically
2. Logged in as that superuser, go to some other user that has some permissions already set.
3. Add/delete permissions for that user.
4. See that the permission list is correctly updated once the page is reloaded and in the backend.

#### Link(s) to related issue(s)

* Resolves #7657
